test/functional/ignore_undefined.test.js

```diff
@@ -148,4 +149,132 @@ describe('Ignore Undefined', function () {
       });
     }
   });
+
+  it('Should correctly inherit ignore undefined field from db during insert', {
+    metadata: { requires: { topology: ['single'] } },
```

Comment:
Is there a reason to only test against a standalone? If you want to test on any topology type/server version then you can omit the `metadata` completely:
```
if('should correctly inherit ignoreUndefined from db during insert', function(done) {
  ...
});
```